src/reprepro_updater/helpers.py

```diff
@@ -92,10 +94,13 @@ def invalidate_packages(repo_dir, distro, arch, packages):
         return True
     filterstr = 'Package (== {})'
     filterlist = [filterstr.format(pkgname) for pkgname in packages]
-    invalidate_packages_command = ['reprepro', '-b', repo_dir,
-                                   '-T', 'deb', '-A', arch, '-V',
-                                   'removefilter', distro,
-                                   ' | '.join(filterlist)]
+    cmd_prefix = ['reprepro', '-b', repo_dir, '-T', 'deb',
+                  '-A', arch, '-V', 'removefilter', distro]
+    invalidate_packages_command = cmd_prefix + [' | '.join(filterlist)]
+    if len(invalidate_packages_command) > ARG_MAX_LEN:
```

Comment:
Won't this just give you the length of the array?

Reply:
Yes it will. Shows what comes of rushing. I can't recall if SC_ARG_MAX applies to an individual arg or the entire line.

I'll visit the man pages and revise this.